bsdlib: version 0.7.7 (#231)
nrfconnect/sdk-nrfxlib: bsdlib/doc/CHANGELOG.rst

All notable changes to this project are documented in this file.

bsdlib 0.7.7
************

> * Fixed a bug introduced in version 0.7.5 in bsd_init() that would leave the library in an inconsistent state when updating the modem firmware.

Reviewer: ```suggestion
* Fixed a bug in bsd_init() (introduced in the version 0.7.5) that caused the library to be in an inconsistent state when updating the modem firmware.
```

Author: Thanks, applied.
